app/controllers/analytics/facilities_controller.rb

```diff
@@ -2,7 +2,11 @@ class Analytics::FacilitiesController < AnalyticsController
   before_action :set_facility
 
   def show
-    @analytics = Rails.cache.fetch(analytics_cache_key) { analytics }
+    if ENV.fetch('SIMPLE_SERVER_ENV') == 'qa'
```

Comment:
Same here.

Reply:
@govindkrjoshi Added new env variable ENABLE_CACHED_QUERIES_FOR_DASHBOARD. Toggling on the same.